awswrangler/s3/_read_parquet.py

```diff
@@ -279,7 +279,39 @@ def _arrowtable2df(
     return df
 
 
-def _read_parquet_chunked(
+def _pyarrow_chunk_generator(
+    pq_file: pyarrow.parquet.ParquetFile,
+    chunked: Union[bool, int],
+    columns: Optional[List[str]],
+    use_threads_flag: bool,
+) -> Iterator[pa.RecordBatch]:
+    if chunked is True:
+        batch_size = 65_536
```

Comment:
I assume this is the default from the `iter_batches` method 👍🏼

Reply:
Yes it's the `pyarrow` default 👍